aep/general/0124/aep.md.j2

```diff
@@ -1,5 +1,86 @@
 # Resource association
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/57) for more
-information.
+APIs sometimes have resource hierarchies that can not be cleanly expressed in
```

Comment:
```suggestion
APIs sometimes have resource relationships that can not be cleanly expressed in
```

I think "hierarchies" implies parent-child relationships - while these are commonly use for relationships.